slatedb/src/oracle.rs

```diff
@@ -48,4 +62,14 @@ impl Oracle {
             ..self
         }
     }
+
+    pub(crate) fn track_last_committed_seq(&self, seq: u64) {
```

Comment:
How will this work when we exit uncleanly and have to replay the wal. Those writes don't go through this path. We could write them to the tracker from the replay path, but how would we know the timestamp to use? 🤔

Reply:
One option might be to fill the gap using the last timestamp the tracker is aware of

Reply:
Hmm that's a really good catch... I should probably work that into the sequence tracker to ensure that there are no gaps. I'll fill in gaps with the last timestamp and add a test